sync/src/main/java/tech/pegasys/teku/sync/forward/multipeer/SyncController.java

```diff
@@ -88,12 +103,14 @@ public void onTargetChainsUpdated() {
 
   private void onSyncComplete(final SyncResult result) {
     eventThread.checkOnEventThread();
+
     if (isSyncActive() || result == SyncResult.TARGET_CHANGED) {
       // A different sync is now running so ignore this change.
       LOG.debug(
           "Ignoring sync complete because another sync is already active. Result: {}", result);
       return;
     }
+    eventLogger.syncCompleted();
```

Comment:
I think this is potentially too soon - we may wind up starting a new sync to a different target just below.  I suspect we want to move this to where we call `notifySubscribers(false)` just below.